### PR TITLE
updates for removing the router

### PR DESCRIPTION
**Pull request by jinhoonbang**

### CODE REVIEW

**Comment by vreff** [comment]
Lot of routine copy/paste/delete, it looks good to me, so I'm just gonna go off of CI results on this.

File: integration-tests/contracts/contract_deployer.go [hunk 1]
```diff
@@ -3,10 +3,11 @@ package contracts
 import (
 	"errors"
 	"fmt"
-	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/generated/automation_consumer_benchmark"
 	"math/big"
 	"time"
 
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/generated/automation_consumer_benchmark"
```
Comment (jmank88) [inline integration-tests/contracts/contract_deployer.go:9]: nit/ this would fit in nicely down below

**Review by makramkd — COMMENTED** [review]
Does this require a new version of ocr2vrf?

**Comment by jinhoonbang** [comment]
> Does this require a new version of ocr2vrf?

yep. updated now

**Review by makramkd — APPROVED** [review]

**Pull request merged**